src/stan-users-guide/regression.Rmd

```diff
@@ -481,34 +481,34 @@ parameter block would be redefined to use $(K - 1)$-vectors
 
 ```stan
 parameters {
-  matrix[K - 1, D] beta_raw;
+  matrix[D, K - 1] beta_raw;
 }
 ```
 
 and then these are transformed to parameters to use in the model.
 First, a transformed data block is added before the parameters block
-to define a row vector of zero values,
+to define a vector of zero values,
 
 ```stan
 transformed data {
-  row_vector[D] zeros = rep_row_vector(0, D);
+  vector[D] zeros = rep_vector(0, D);
 }
 ```
 
-which can then be appended to `beta_row` to produce the
+which can then be appended to `beta_raw` to produce the
 coefficient matrix `beta`,
 
 ```stan
 transformed parameters {
-  matrix[K, D] beta;
-  beta = append_row(beta_raw, zeros);
+  matrix[D, K] beta;
```

Comment:
This could be

```
matrix[D, K] beta = append_col(beta_raw, zeros);
```

Reply:
Thanks for pointing that out. Fixed now.